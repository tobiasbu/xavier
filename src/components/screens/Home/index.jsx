import React from 'react';
import Content from '../../container/Content';
import Header from '../../container/Header';
import Title from '../../basic/Title';

/**
 * Magnetos home page.
 */
const Home = () => {
  const total = 0;
  return (
    <Content>
      <Header>Seja bem vindo!</Header>
      <Title size={4}>{`VocÃª tem o total de ${total}`}</Title>
    </Content>
  );
};

export default Home;
