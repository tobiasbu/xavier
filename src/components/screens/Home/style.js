
import { createUseStyles } from 'react-jss';

const useStyle = createUseStyles({
  mainBox: {
    margin: 0,
    padding: '2rem',
    background: 'var(--color-space-200)',
    borderRadius: 8,
  },
});

export default useStyle;
