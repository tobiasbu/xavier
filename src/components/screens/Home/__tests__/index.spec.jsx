import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Home from '..';

describe('Home screen', () => {
  it('should render', () => {
    const component = shallow(
      <Home />,
    );
    expect(toJSON(component)).toMatchSnapshot();
  });
});
