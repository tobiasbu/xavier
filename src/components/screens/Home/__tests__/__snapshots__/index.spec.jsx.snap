// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home screen should render 1`] = `
<Content>
  <Header>
    Seja bem vindo!
  </Header>
  <Text
    className=""
    size={1}
  >
    <span
      aria-label="high voltage"
      role="img"
    >
      ⚡
    </span>
    <strong>
       xavier 
    </strong>
    é muito mais do que um programa de gestão financeira.
  </Text>
  <Text
    className=""
    size={1}
  >
    <span
      aria-label="high voltage"
      role="img"
    >
      ⚡
    </span>
    <strong>
       xavier 
    </strong>
    fornece um espaço para seus usuários com soluções ágeis e fáceis, para qualquer tipo de necessidade.
  </Text>
  <Text
    className=""
    size={1}
  >
    Comece agora já a usar a melhor plataforma de gestão financeira, gratuitamente!
  </Text>
</Content>
`;
